04-onion-routing.md

```diff
@@ -823,9 +823,8 @@ channel.
 ### Requirements
 
 The _erring node_:
-  - SHOULD set `pad` such that the `failure_len` plus `pad_len` is equal to 256.
-    - Note: this value is 118 bytes longer than the longest currently-defined
-    message.
+  - SHOULD set `pad` such that the `failure_len` plus `pad_len` is equal to
```

Comment:
Would a less disruptive change be to say that it should pad the message modulo `256`? Such that in most cases (existing failures) would still be 256 without breaking nodes that do a hard check on the length.

Reply:
Maybe a good idea if we don't do the signaling. But it does reveal info to intermediate nodes about the type of failure possibly.